CHANGELOG-3.8.md

```diff
@@ -201,11 +201,10 @@ C++ compiler with support for std=c++17 or higher.
   auto widget = communicator->propertyToProxy<WidgetPrx>("MyWidget");
   ```
 
-- The C++ structs and classes generated by the Slice compiler can now be printed using
-`operator<<(ostream&, const T&)`. This operator prints the type name and all the field names and values; see the
-[Slice/print] test for examples.
-  - You can also implement your own custom printing by applying metadata `["cpp:custom-print"]` to your Slice struct or
-    class.
+- The C++ structs, classes, exception classes and enumerations generated by the Slice compiler can now be printed using
```

Comment:
Saying "classes, exception classes" feels a little redundant to me.
Also, missing oxford comma between "exception classes and enumerations"